config/webhook/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+resources:
```

Comment:
Assuming this was added for local dev purposes, please don't forget to remove it.